control/frdata.py

```diff
@@ -173,7 +166,7 @@ def __str__(self):
                 outstr.extend(
                     ['%12.3f  %10.4g%+10.4gj' % (w, m, p)
                      for m, p, w in zip(real(self.fresp[j, i, :]),
-                                        imag(self.fresp[j, i, :]), wt)])
+                                        imag(self.fresp[j, i, :]), self.omega)])
```

Comment:
```suggestion
                                        imag(self.fresp[j, i, :]),
                                        self.omega)])
```
style

Reply:
or even better turn the order of zip elements around to match the `w, m, p` order of the format string

Reply:
done. even better than that: make the letter mnemonic match what it is actually printing (real/imag rather than mag/phase)